setup.py

```diff
@@ -49,6 +49,7 @@ def _read_requirements():
             'data_registry_post = data_pipeline_api.registry.upload:upload_cli',
             'data_registry_create_model = data_pipeline_api.registry.create_model:create_model_cli',
             'data_registry_upload_input = data_pipeline_api.registry.upload_input:upload_input_cli',
+            'convert_access_to_config = scripts.convert_access_to_config:convert_cli',
```

Comment:
maybe we should agree on a common prefix for all of these things, RR has in the past called things `pipeline_x` when he talks about the various CLIs

Reply:
Yep seems sensible, should I change them all now?

Reply:
sure